components/src/themed/ThemedButton.tsx

```diff
@@ -39,7 +39,8 @@ export const ThemedButton: React.FC<ThemedButtonProps> = (props) => {
                 textColor = theme.colors.textPalette.disabled;
             } else {
                 backgroundColor = theme.colors.primaryPalette.light;
-                textColor = theme.colors.textPalette.highlight;
+                textColor =
+                    theme.colors.overrides.button?.contained?.text?.disabled || theme.colors.textPalette.disabled;
```

Comment:
Shall we add an additional value to fallback to if neither of these custom properties is available in the theme (theme.colors.text)? Or are we assuming that anybody using these components will be using a version 6 theme?